Jupyter_notebook/tutorial.ipynb

```diff
@@ -240,9 +252,9 @@
       },
       "outputs": [],
       "source": [
-        "files = glob.glob(os.path.join(path, 'CCF/*.h5'))\n",
+        "files = glob.glob(os.path.join(stack_data_path, '**/*.h5'))\n",
         "print(files)\n",
-        "plotting_modules.plot_all_moveout(files[0], 'Allstack_linear', 0.1, 0.2, 'ZZ', 1)\n"
+        "plotting_modules.plot_all_moveout(files, 'Allstack_linear', 0.1, 0.2, 'ZZ', 1)"
```

Comment:
Looks like `plot_al_moveout` is expecting the first argument to be a list of files. Still, this failed with the `-1` error like before.